tools/src/main/java/org/apache/kafka/tools/VerifiableProducer.java

```diff
@@ -242,66 +207,174 @@ public String getValue(long val) {
     /** Close the producer to flush any remaining messages. */
     public void close() {
         producer.close();
-        System.out.println(shutdownString());
+        printJson(new ShutdownComplete());
     }
 
-    String shutdownString() {
-        Map<String, Object> data = new HashMap<>();
-        data.put("name", "shutdown_complete");
-        return toJsonString(data);
+    @JsonPropertyOrder({ "timestamp", "name" })
```

Comment:
I like the `JsonPropertyOrder` solution when it's being applied on top of an existing class. But overall this patch makes things more verbose (we've added almost 100 lines). That's not necessarily a problem, but would it maybe be simpler to just change the existing maps to be `LinkedHashMap`s so they preserve their order and then order the `put` calls the way we want them?